src/Application/Migrations/__2019-09-29_skip_dependent.sql

```diff
@@ -0,0 +1,184 @@
+BEGIN;
+	
+SET ROLE TO postgres;
+
+ALTER TABLE tbl_project_ticket_state
+  ADD COLUMN skip_on_dependent boolean NOT NULL DEFAULT false;
+
+ALTER TABLE tbl_ticket_state
+  ADD COLUMN skippable_on_dependent boolean NOT NULL DEFAULT false;
```

Comment:
That column only exists in the migration patch?!

Reply:
And it only seems to be used when displaying the states, but a database user would be allowed to set that flag on states  not marked as skippable? or did I miss something

Reply:
> And it only seems to be used when displaying the states, but a database user would be allowed to set that flag on states not marked as skippable?

It belongs to the state, so it's stored together with states. I'm not sure we need additional validation if the state marked to skip in a project is skippable?

Reply:
> That column only exists in the migration patch?!

Fixed, thanks.